library/CHART-TEMPLATE/tests/common_library_lowdatamode_test.yaml

```diff
@@ -0,0 +1,52 @@
+suite: test that lowdatamode works as expected
+templates:
+  - templates/config-map-of-examples.yaml
+release:
+  name: my-release
+  namespace: my-namespace
+set:
+  licenseKey: test-license-key
+  cluster: test-cluster
+tests:
+  - it: By default it should be false
```

Comment:
Mostly a nit, but typically test names/cases are written in plain present tense, e.g.
> Is false by default
> Overrides global with local

Rather than

> Should be false by default
> Allow to override (to true) the global state locally

Reply:
Good to know.

Solved in [9b9b76e](https://github.com/newrelic/helm-charts/commit/9b9b76e3ccfbf000303049efe883be4483698ad4)